spec/requests/api/v1/documents_spec.rb

```diff
@@ -2,55 +2,60 @@
 require 'uri'
 
 describe API::V1::Documents do
-  let(:id) { "some really!weird@id.name" }
-  let(:valid_params) do
```

Comment:
Is there are a reason you removed the `valid_params` block out of the main `describe` block? I would think we are using that throughout the specs...?

Reply:
I noticed that only Post was using it. Everywhere else is creating its own. To keep it consistent with all the other test cases, I moved it to  'POST /api/v1/documents'.